po/ka.po

```diff
@@ -0,0 +1,23 @@
+# Georgian translations for add-to-desktop package.
+# Copyright (C) 2020-2025 the add-to-desktop's copyright holder
+# This file is distributed under the same license as the add-to-desktop package.
+# Ekaterine Papava <papava.e@gtu.ge>, 2025.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: add-to-desktop@tommimon.github.com\n"
+"Report-Msgid-Bugs-To: https://github.com/Tommimon/add-to-desktop\n"
+"POT-Creation-Date: [DATE_TIME]\n"
+"PO-Revision-Date: 2025-12-11 07:03+0100\n"
+"Last-Translator: Ekaterine Papava <papava.e@gtu.ge>\n"
+"Language-Team: Georgian\n"
+"Language: ka_GE\n"
```

Comment:
Language code should match file name. I think here you can put `ka` instead of `ka_GE` and we can skip on the country specification. We can assume this translation is valid for all Georgian speakers unless you think otherwise.